### DISCUSSION TITLE
New links criteria

### DISCUSSION THREAD

**Discussion by GeorgeSG** [GeorgeSG/sub.rehab#2] [General]
👋 Hello folks,

First I wanna say thanks to everybody who's submitting links and helping me keep the website up to date! 🙇‍♂️

I've been thinking about some requirements for adding new links. In the past days, I've basically added every suggestion that seemed legit. I have been trying to vet things, but there's been more and more submissions and I realize there should be some transparent criteria.

### Going forward I suggest the following criteria:
1. No links to communities who break their service's ToS or share any illegal content (this should be self-explanatory).
1. **Link must be an alternative to an existing subreddit.** I don't see sub.rehab as a general fediverse directory (for now).
1. **Community must have a minimal amount of activity.** I am thinking - at least **3** posts that are not submitted by the community's moderator team.
    - This does not apply for communities that are verified and proven as official replacements of the original subreddit

I'm planning to keep all the links I've added so far and start using these criteria for new suggestions going forward.

Let me know what you think below!

**Comment by stephen304** [comment]
@GeorgeSG 

It would be great to allow amendment submissions to existing ones. I am trying to submit the proof for https://lemmy.dbzer0.com/c/piracy being official but it rejects it since it's already submitted as unofficial (https://www.reddit.com/r/Piracy/comments/14dg6al/from_now_on_only_sexy_pirate_john_oliver_artwork/)

Same for https://startrek.website/c/startrek (https://www.reddit.com/r/startrek/comments/14aifzm/rstartrek_reddit_and_the_future/)

**Reply by GeorgeSG** [reply]
👍  agreed. Added to the to do

**Reply by GeorgeSG** [reply]
I'm tracking this in #8

**Reply by GeorgeSG** [reply]
@stephen304 Updated both links to be marked official. For now, I'm starting a discussion thread for link reporting [here](https://github.com/GeorgeSG/sub.rehab/discussions/9).

If there's a lot of demand for reporting/editing links, I'll add it as a feature in the site itself.

**Comment by YoshiWalsh** [comment]
Would you consider adding raddle.me as a service? A lot of queer, progressive, and leftist spaces seem to have found new homes there. It's also pretty active.

E.g.
 - https://raddle.me/f/196
 - https://raddle.me/f/egg_irl
 - https://raddle.me/f/LateStageCapitalism
 - https://raddle.me/f/trans
 - https://raddle.me/f/traaaaaaannnnnnnnnns

It also provides alternatives to many other popular subreddits (programming, music, anime, etc...)

**Reply by GeorgeSG** [reply]
I am accepting raddle.me links in the "misc" category for now. There is currently one link.

i don't have a strict criteria for when a service gets it's dedicated category tag for now. If there is a sizable amount of links, I'll add it as it's own category.

**Reply by GeorgeSG** [reply]
@JoshuaWalsh After the latest set of link additions, raddle is a standalone service now.

**Comment by neurozero** [comment]
What's the definition of official?

**Reply by nick-w-nick** [reply]
I think instead of trying to decide if something is "official" or not, we should go off of the amount of activity in the alternative community. Anyone could just be the first person to submit a link to an alternative, but the link may be to a community that no one participates in.

How we keep track of that metric is up for discussion. If we nailed down a list of supported communities, we could then create an integration for each one that updates the activity metrics on a regular basis and automatically reorders the links for each community based on those values. Not to mention this would be a great metric to be able to display on the sub.rehab page for each entry.

Of course, that is a level of overhead that some may not be interested in maintaining, but I see it as a fairly straightforward implementation, especially since a majority of the currently supported communities are part of the fediverse, and already have mostly-sufficient plug-and-play integration utilities.

**Reply by GeorgeSG** [reply]
The current criteria I go by is - the link is listed in any of:
- the subreddit's sidebar
- in a pinned thread
- thread started by a moderator of the subreddit
- in the sub privated message

**Reply by PrayagS** [reply]
@GeorgeSG Thanks for clarifying the criteria.

I submitted a request yesterday for [r/adhdindia](https://reddit.com/r/adhdindia)'s Discord server. Didn't go through apparently.

I did mention the reason that the link is pinned in the sidebar. Was there some other reason this got missed?

**Reply by GeorgeSG** [reply]
@PrayagS 

Could be one of these:
- When I'm approving links I'm trying to go fast, so could be something I missed in a hurry;
- I did not manage to get through all suggestions :) 
- I browse reddit through old.reddit.com and I just checked - can't see the discord link there (I can see it from reddit.com though)

**Reply by PrayagS** [reply]
@GeorgeSG Ah I see :)

And yes, our subreddit is relatively new so we don't maintain the old Reddit appearance. Thanks for getting back. Let me know if there are any other issues with verification.

**Reply by namesarebloat** [reply]
@GeorgeSG Under your criteria raddle.me/f/196 should be official
https://www.reddit.com/r/196/comments/146y2qw/r196_offsite_migration_discord_info/

**Reply by GeorgeSG** [reply]
@namesarebloat Addresed. Thank you!

**Comment by sirvalkyerie** [comment]
Does Pillowfort not count for a misc link? They use forums with comments and upvotes and the like. It's a lot more reddit-like than Discord.

**Reply by GeorgeSG** [reply]
@sirvalkyerie We have some pillowfort links submitted, but we're currently holding onto them.

I'm not very familiar with the platform. At a first glance it seems more like twitter / instagram than a discussion-based forum. I find it confusing that people are "reblogging" posts in topics, is it more like everyone has their blog?

Do you mind explaining briefly how it works and how do communities form there? E.g. how would something like, say r/shittyfoodporn form there, be moderated, and be discovered?

**Reply by GeorgeSG** [reply]
Also, I'm not sure how access works. I can browse, but I have to pay to post?

**Reply by sirvalkyerie** [reply]
So it's superficially like Tumblr but functions more like Reddit. You have a profile and you can post to your own profile (just like on Reddit how you can post to your own profile now too) and it looks like Tumblr. But aside from that it's like Reddit. Each community is self-contained in its own subreddit style instance and people can comment and upvote on them and favorite them etc. You join a community just like a subreddit. Instead of something like Tumblr where it's just a big expansive blogsite. 

So for instance there's a Pokemon community. Just like /r/Pokemon you can post to it and comment on posts and like posts. But you have to be a member of that community to do so. It's not like Tumblr. Here's a screenshot of a comment chain on a post in the pillowfort pokemon community that shows you a treestyle just like Reddit/Lemmy/Kbin etc. 
![image](https://github.com/GeorgeSG/sub.rehab/assets/57422681/7ab419b2-d87c-4e68-91fc-732ff117f019)


As far as joining Pillowfort, there's a waitlist. Right now it's only like 90 minutes to get your invite. But if you want to you can pay to get your invite right away. It's to keep the site from crashing as people rush to it. But you don't have to pay to post or to register. It's just to jump to the front of the line for now. 

You can log into Pillowfort, go to create a community, create shittyfoodporn, assign other moderators to it and then people can start joining and posting to it. Just like Reddit or Lemmy etc. It's basically Reddit but with a Tumblr-like interface.

**Reply by GeorgeSG** [reply]
@sirvalkyerie thank you very much for the detailed explanation! This is very useful!

I will consult with the team and we'll make a decision. I'll update you if we decide to start listing pillowfort links.
